Modules.md

```diff
@@ -37,18 +37,21 @@ the host environment to include a JavaScript VM.
 
 A module defines a set of functions in its
 [code section](Modules.md#code-section) and can declare and name a subset of
-these functions to be **exports**. The meaning of exports (how and when they are
-called) is defined by the host environment. For example, a minimal shell
-environment might only probe for and call a `_start` export when given a module
-to execute.
+these functions to be **exports**. A module can also a declare and name a
+subset of its [globals](AstSemantics.md#global-variables) to be exports.
+The meaning of exports (how and when they are called) is defined by the host
+environment. For example, a minimal shell environment might only probe for and
+call a `_start` export when given a module to execute.
 
 A module can declare a set of **imports**. An import is a tuple containing a
-module name, the name of an exported function to import from the named module,
-and the signature to use for that import within the importing module. Within a
-module, the import can be [directly called](AstSemantics.md#calls) like a
-function (according to the signature of the import). When the imported
-module is also WebAssembly, it would be an error if the signature of the import
-doesn't match the signature of the export.
+module name, a name exported by the named module, and the type
+to use for the import within the importing module. Within a module, imported
+functions can be [directly called](AstSemantics.md#calls) just like normal
+functions (according to the signature of the import). Similarly, imported
+globals can be [directly accessed](AstSemantics.md#global-variables) just like
```

Comment:
Direct access seems reasonable for intra-wasm, but did you have a use case in mind for it specifically?